hoomd/ParticleGroup.h

```diff
@@ -315,6 +315,16 @@ class PYBIND11_EXPORT ParticleGroup
         */
         unsigned int intersectionSize(std::shared_ptr<ParticleGroup> other);
 
+        /** Thermalize the particle momenta
+
+            Draw gaussian random momenta for all particles in the group.
```

Comment:
```suggestion
            Draw Gaussian random momenta for all particles in the group.
```
My personal preference, I don't know that capitalization is really standard but it's more common in my experience.

Reply:
Yes it is standard, I don’t know why my spell checker didn’t catch it.